src/std/build-spec.ss

```diff
@@ -21,9 +22,8 @@
     (gxc: "error" ,@(include-gambit-sharp))
     "getopt"
     "logger"
-    (gxc: "generic/dispatch" ,@(include-gambit-sharp))
+    "generic/dispatch"
     "generic/macros"
-    (gxc: "generic/builtin" ,@(include-gambit-sharp))
```

Comment:
Is include-gambit-sharp now implicitly there everywhere?

Reply:
no, this module is just gone.